src/app/pages/MediaArticlePage/PagePromoSections/LastestMediaSection/LatestMediaIndicator/index.tsx

```diff
@@ -0,0 +1,32 @@
+/** @jsxRuntime classic */
+/** @jsx jsx */
+import { jsx } from '@emotion/react';
+import moment from 'moment';
+import formatDuration from '#app/lib/utilities/formatDuration';
+import { LatestMediaIndicatorProp } from '../LatestMediaTypes';
+
+import { StyledPromoMediaIndicator, styles } from './index.styles';
+
+const LatestMediaIndicator = ({ duration }: LatestMediaIndicatorProp) => {
+  const momentDuration = moment.duration(duration, 'seconds');
+  const durationString = formatDuration({ duration: momentDuration });
+  const isoDuration = momentDuration.toISOString();
+
+  return (
+    <div css={styles.placeholderInfo}>
+      <StyledPromoMediaIndicator />
```

Comment:
Could we use `Promo.MediaIndicator` here and pass in the css using the CSS prop, like `css={styles.promoMediaIndicator}`. I think that would result in the same output.